dali/image/png.cc

```diff
@@ -36,6 +37,11 @@ DALIError_t GetPNGImageDims(const uint8 *png, int size, int *h, int *w) {
 
   // IHDR needs to be the first chunk
   const uint8 *IHDR = png + 8;
```

Comment:
Should there be a bounds check on size>8?

Reply:
Done